src/roman_datamodels/stnode/_schema.py

```diff
@@ -14,7 +14,7 @@
     SCHEMA_URIS_BY_TAG,
 )
 
-__all__ = []
+__all__ = ["Builder"]
```

Comment:
Why is this being added to the public API?

Reply:
Why not add a `Builder` protocol? 

```python
from typing import Protocol

class Builder(Protocol):

    def build(self, schema: Mapping[str, Any], defaults: Any | None = None):
         """Method to build a map based on a ``schema`` with values overridden by the passed``defaults``""""

```

Reply:
I did not add any method called `create_from_node`. That was added by you in #550 so it was already public API. As far as I am aware I didn't actual "add" or even change public API with this PR as the Protocol is simply a description of what was expected to be passed to a public function already.

Reply:
Thanks. Yeah that was my bad on `create_from_node` I lost track of that change with the addition of `_TaggedNodeMixin` in this PR.

I do not think `Builder` should be public. If that's a sticking point for this PR we can put this on hold until the arguments to the `create_*` functions can be renamed to `_builder`.